hack/ci/e2e.sh

```diff
@@ -106,9 +106,17 @@ apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
 # the control plane node
 - role: control-plane
+  extraMounts:
+  - containerPath: /etc/kubernetes/audit-policy.yaml
+  - hostPath: /tmp/audit-policy.yaml
 - role: worker
 - role: worker
 EOF
+    # create the audit-policy
+    # https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy
+    curl https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/gce/gci/configure-helper.sh > /tmp/helper.sh
+    . tmp/helper.sh
+    create-master-audit-policy /tmp/audit-policy.yaml
```

Comment:
hm, my vote would be to not use the the k/k 's `/cluster` tools.
if not downloading it from another location as ready to use YAML, one alternative is to put the yaml in the same folder as this bash script and access it with `dirname "$0"` (or similar).

Reply:
Sounds good.
Should I submit a separate PR for test-infra/experiment/?

Reply:
not sure. i would defer to @BenTheElder on test-infra/experiment/

Reply:
yes, let's not depend on cluster/.. why do we need the configure-helper?

we shouldn't use test-infra/experiment for kind.

Reply:
we have our own hack dir, and cloning test-infra is a heavy requirement (> 160MB!!)